hubblestack/utils/osquery_lib.py

```diff
@@ -0,0 +1,41 @@
+"""
+HubbleStack osquery lib. Can be used to execute osquery queries from Hubble code
+Author - Mudit Agarwal (muagarwa@adobe.com)
+"""
+import logging
+import os
+import salt.modules.cmdmod
+import json
+
+__salt__ = {'cmd.run': salt.modules.cmdmod._run_quiet,
+            'cmd.run_all': salt.modules.cmdmod.run_all}
+
```

Comment:
Just add
```
global __grains__
```
here?

Reply:
This module is used as an import in one the docker_details grain. Since __grains__ is not yet defined, it cannot be used here.